src/blob_gc_picker.cc

```diff
@@ -57,8 +57,10 @@ std::unique_ptr<BlobGC> BasicBlobGCPicker::PickBlobGC(
         stop_picking = true;
       }
     } else {
-      if (blob_file->GetDiscardableRatio() >=
-          cf_options_.blob_file_discardable_ratio) {
+      if (blob_file->file_size() <= cf_options_.merge_small_file_threshold ||
```

Comment:
Does it mean we are more aggressively merging small files than before?

Reply:
Yep, merging small file doesn't work as expected before cause `DoSample` doesn't consider `merge_small_file_threshold`.

Reply:
I see.